csharp/src/Google.Protobuf.Test/Google.Protobuf.Test.csproj

```diff
@@ -6,6 +6,7 @@
     <SignAssembly>true</SignAssembly>
     <IsPackable>False</IsPackable>
     <AllowUnsafeBlocks>True</AllowUnsafeBlocks>
+    <LangVersion>10.0</LangVersion>
```

Comment:
This shouldn't normally be needed, given the Directory.build.props in the parent directory. Was it causing a failure with the Conformance tests, because now they're being built out of a separate directory structure?

Reply:
I didn't make the connection that moving it to a separate directory could have been breaking this.  Lemme see if I can remove it after moving it back

Reply:
Ah the problem was that I wasn't including Directory.build.props in the Bazel rule.  Bazel sandboxes all builds, so it won't see files you don't provide it :)